### ISSUE TITLE
Combine with mediapipe selfie segmentation

### ISSUE THREAD

**Issue by defrindr** [hiukim/mind-ar-js#552]
Can anyone give me an example of how to use mind-ar-js with mediapipe selfie segmentation?

**Comment by mpalenque** [comment]
Yeah, here. but is not working on ios. mediapipe+mindar has some problems on ios safari or chrome @hiukim 

https://codepen.io/mpalenque/pen/qBeGwPQ
any idea how to fix it ? I tried using mediapipe video stream on front of mindar video stream. works on desktop but not on IOS

also tried using mediapipe texture as material, works with a-frame but as soon as I add mindar, it doesn't work on IOS
example 2:
https://codepen.io/mpalenque/pen/ZEgNdmp
